plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -811,25 +812,33 @@ private function months_filter() {
 	protected function get_months_filter_options(): array {
 		global $wpdb;
 
+		$this_month = new \WC_DateTime(
+			sprintf(
+				'%s-%s-01',
+				gmdate( 'Y' ),
+				gmdate( 'm' )
+			),
+			wp_timezone()
+		);
+
 		$orders_table = esc_sql( OrdersTableDataStore::get_orders_table_name() );
 		$trash_status = esc_sql( OrderStatus::TRASH );
 
 		$first_year_month_gmt = $wpdb->get_row(
-			// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
 			$wpdb->prepare(
-				"
+				'
 					SELECT YEAR( t.date_created_gmt ) AS year,
 					       MONTH( t.date_created_gmt ) AS month
-					FROM $orders_table t
+					FROM %i t
```

Comment:
`%i` sadly doesn't work with WPCOM's wpdb, so I guess we shouldn't rely on it.

Reply:
Dangit, I should have known there was a reason we weren't already using it. Fixed in 5c9aa738df